doc/nrf/device_guides/working_with_nrf/nrf70/features.rst

```diff
@@ -43,9 +43,12 @@ Currently, the nRF70 Series devices support the following modes:
 * :ref:`Software-enabled Access Point (SoftAP or SAP) <nRF70_soft_ap_mode>`: Operates as a virtual access point device.
 * :ref:`Scan <ug_nrf70_developing_scan_operation>`: Operates as a scan-only device.
 * :ref:`Radio test <wifi_radio_test>`: For PHY (Baseband and Radio) characterizations and calibrations.
+* :ref:`Monitor <ug_nrf70_developing_raw_ieee_80211_packet_reception>`: Operates as an IEEE 802.11 wireless packet sniffer.
 
-The nRF70  Series devices also support the functionality of :ref:`ug_nrf70_developing_raw_ieee_80211_packet_transmission`.
-This allows the injection of raw 802.11 frames in any of the above modes.
+The nRF70 Series devices also support the following functionalities:
+
+* :ref:`ug_nrf70_developing_raw_ieee_80211_packet_transmission`: Allows the injection of raw IEEE 802.11 frames in Station and Monitor modes.
+* :ref:`Promiscuous reception <ug_nrf70_developing_promiscuous_packet_reception>`: Allows the reception of IEEE 802.11 packets from a connected BSSID when operating in Station mode.
```

Comment:
@sachinthegreen @VivekUppunda Please confirm, but shouldn't "Promiscuous mode" be added to the "Support modes" section above as well (as in, after line 46 of this doc)?

I see it was added to the changelog as a mode as well in a [previous PR](https://github.com/nrfconnect/sdk-nrf/pull/13896/files):
![image](https://github.com/nrfconnect/sdk-nrf/assets/51916561/6d97a66a-533e-4436-b5cb-9a4d1f7c5d51)

Reply:
@kittydepa - Promiscuous mode and TX injection mode are not primary modes of operation. They operate under primary modes of operation which at present are Monitor and Station modes. I think this is okay though